x-pack/legacy/plugins/transform/public/app/services/http_service.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// service for interacting with the server
+
+import chrome from 'ui/chrome';
+
+// @ts-ignore
+import { addSystemApiHeader } from 'ui/system_api';
```

Comment:
Should this come via the shim?

Reply:
I'll look into this in a follow up. #45352 has already a note on consolidating the somewhat mixed use of multiple `http` approaches in the whole plugin now.